app/forms/base_guide_form.rb

```diff
@@ -128,8 +128,9 @@ def next_author_id
   end
 
   def promote_errors_for(model)
+    ignored_attributes = [:editions]
     model.errors.each do |attrib, message|
-      errors.add(attrib, message)
+      errors.add(attrib, message) unless ignored_attributes.include? attrib
```

Comment:
How about passing in which attribute(s) to ignore as a method argument? Something like..

```
def promote_errors_for(model, opts = {})
  ignored_attributes = opts.fetch(:ignored_attributes, [])
```